test/cli/autocorrect-extend/test.out

```diff
@@ -6,6 +6,7 @@ def top_level; end
 
 # Should only add one extend
 class SuggestSigAndExtend
+  extend T::Sig
```

Comment:
Either this comment is wrong, or the code in `AutocorrectSuggestion.cc` is wrong.